_posts/2020-11-09-the-cloud-data-warehouse.md

```diff
@@ -0,0 +1,158 @@
+
+Many important questions arise when people consider migrating to Cloud Data Warehouses, such as
+
+* How to transition skilled database administrators and operations staff to cloud?
+* What extra skills are required with a Cloud Data Warehouses?
+* Is a Cloud Data Warehouse going to lower our TCO for Data Warehousing?
+
+These are complicated questions to answer but it's worth bearing in mind that the effort required to maintain business services doesn't go away, it just changes in nature. As we'll explore in the next article, the core skills of ETL developers are absolutely transferable.
+
+Some new themes emerge with Cloud Data Warehouses; though the technology or implementations may change, the concerns and risk controls will typically be very familiar to experienced data warehouse operations professionals.
+
+### FinOps
+
+[FinOps](https://www.finops.org/what-is-finops/) is concerned with prudent financial management in the Cloud. This is a major sea change for most large organisations that are used to centralising management and procurement of IT hardware. The Cloud Native way is to bring accountability back to teams both for their spend and management of the resources they provision.
+
+To make FinOps successful you need strong, centralised compliance and guardrail frameworks in place to ensure common baseline behaviours and expectations, such as tagging resources with cost codes, enacting soft limits based on policy, etc.
+
+In the case of Snowflake, as an example, the following questions emerge that FinOps should set out to address:
+* How do I configure and align cost accountability for users, teams and projects across our organisation?
+* How do I prevent overspend at a team, project or organisation level?
+* How can I forecast my credit usage and plan capacity purchase?
+* How can I rank each Virtual Warehouse in terms of its efficiency to identify wastage?
+
+### SecOps
+
+[SecOps](https://www.sumologic.com/glossary/secops/) applies to all cloud estates including the domain of Cloud Data Warehouses, where the attack vectors for data leakage, pollution and security compromise differ from traditional perimeter secured, on premise data warehouses. Examples of challenges organisations face in this realm are
+* How to appropriately integrate a Cloud Data Warehouse with cloud native access models or external identity providers
+* How to limit network access to specific locations or users or both
+* How to design security hierarchies to prevent leakage of data from new ingress and egress patterns
+* How to report user behaviour and security activity to existing SIEM systems
+
+## Wrapping Up
+
+It's easy to get wrapped up in the promise of sales literature, effervescent customers or case studies, industry hype, and forget that the Data Warehouse is a single part of a longer value chain. A great Cloud Data Warehouse implemented poorly - or strangled by the data acquisition pathways ahead of it - will not turn any ships around. 
```

Comment:
love it.

TBH I want a picture of that value chain at the top of the article. 
and then you can say "CDW's are just a fundamental core part of a modern analytics value chain"